pom.xml

```diff
@@ -1095,6 +1216,16 @@
                 </javaModuleArgs>
             </properties>
             <modules>
+                <module>extensions</module>
```

Comment:
When building with JDK9+ the extensions are built even when the quick profile is set.

Reply:
This is now a non-issue, thanks to @kwart 's expert maven advice: when building with JDK9+ the `modulepath-tests` will be added and a quick build will not include `extensions` and `distribution`.